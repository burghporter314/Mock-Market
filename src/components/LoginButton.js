import React from 'react';
import { useAuth0 } from '@auth0/auth0-react';

const LoginButton = () => {
    const { loginWithRedirect } = useAuth0();

    return(
        <div className="col-lg-12" style={{ "padding": "10px" }}>
            <input onClick={() => loginWithRedirect()} className="btn btn-lg btn-primary btn-block container-fluid" value="Sign In" readOnly></input>
        </div>
    );
}

export default LoginButton;