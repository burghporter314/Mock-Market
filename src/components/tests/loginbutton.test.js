import React from 'react';
import LoginButton from '../LoginButton';
import renderer from 'react-test-renderer';

it('should match snapshot', () => {
    const component = renderer.create(
        <LoginButton/>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
})