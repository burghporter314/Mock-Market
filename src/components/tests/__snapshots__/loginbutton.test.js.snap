exports[`test should match snapshot 1`] = `
<div
  className="col-lg-12"
  style={
    Object {
      "padding": "10px",
    }
  }>
  <input
    className="btn btn-lg btn-primary btn-block container-fluid"
    onClick={[Function]}
    readOnly={true}
    value="Sign In" />
</div>
`;
